nbs/31_text.data.ipynb

```diff
@@ -867,7 +877,29 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
You've got an error here - remember that NBs become docs, so we need them to be clean.

Reply:
Thanks reminder.
Cleared all outputs manully.